docker-compose.yml

```diff
@@ -26,3 +30,13 @@ services:
 
     # Allows accessing dbs through the port forwardings. dev container behaves like host for networking
     network_mode: host
+
+  test_linux:
```

Comment:
How do you feel about having to docker-compose files, one for local development, the other for the Ubuntu runner on CI?

Reply:
Sure, no problem